spec/components/provider_interface/safeguarding_declaration_component_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::SafeguardingDeclarationComponent do
+  let(:heading) { 'Criminal convictions and professional misconduct' }
+  let(:has_shared_text) { 'The candidate has shared information related to safeguarding. Please contact them directly for more details.' }
+  let(:has_not_shared_text) { 'No information shared' }
```

Comment:
These can potentially be translations. Up to you, I'm happy either way.

Reply:
Think I will leave as is as other headings etc. on the page are inline currently.